core/unix/os.c

```diff
@@ -2159,7 +2155,7 @@ get_local_state()
 void
 os_enter_dynamorio(void)
 {
-#    ifdef ARM
+#    if defined(ARM) || defined(RISCV64)
```

Comment:
Please s/ARM/AARCHXX/ as this looks like an oversight from the AArch64 port to stay ARM-only.

Reply:
Done